worlds/witness/options.py

```diff
@@ -62,29 +72,36 @@ class ShuffleDoors(Choice):
 
 
 class DoorGroupings(Choice):
-    """If set to "none", there will be one key for every door, resulting in up to 120 keys being added to the item pool.
-    If set to "regional", all doors in the same general region will open at once with a single key,
-    reducing the amount of door items and complexity."""
+    """
+    If set to "none", there will be one key for every door, resulting in up to 120 keys being added to the item pool.

```

Comment:
I think this number is wrong now, will test later
```suggestion
    If set to "none", there will be one key for each door, resulting in up to 120 keys being added to the item pool.
```